### PR TITLE
work on #259

### PR DESCRIPTION
**Pull request by fabianfrz**
did some work on issue #259

### CODE REVIEW

File: src/www/widgets/widgets/carp_status.widget.php [hunk 1]
```diff
@@ -74,14 +74,14 @@
 }
 if ($ipaddress) {
 ?> &nbsp;
-        <?=htmlspecialchars($status);?> &nbsp;
+        <?=htmlspecialchars(gettext($status));?> &nbsp;
```
Comment (fichtner) [inline src/www/widgets/widgets/carp_status.widget.php]: gettext() requires static strings

Comment (fabianfrz) [inline src/www/widgets/widgets/carp_status.widget.php]: there are three values possoble if you look at the file MASTER, BACKUP, INIT and DISABLED - I will solve that with a switch-case

Comment (fichtner) [inline src/www/widgets/widgets/carp_status.widget.php]: It's not worth fixing for now. Typical chicken-and-egg problems. Until we have a good model that wraps all of this we are stuck here (strings vs. integers for state). :)

**Comment by fabianfrz** [comment]
fixed it

**Comment by fichtner** [comment]
No, this makes it worse, 3 lines now for the same string.. if anyone touches a single string the whole thing breaks.

(a) no translation for that particular bit, or
(b) rework the status completely to return integers and only convert to strings on print (fully translated that is).

In order to move forward (a) is probably a the sensible option.

**Comment by fabianfrz** [comment]
it seems that the code of the widget was wrong the function returning the string translated them and the widget used only hard coded strings so a translation file will break this widget - so we found a bug

**Comment by fabianfrz** [comment]
is there still something wrong?

**Comment by fichtner** [comment]
Looks good now, but will commit tomorrow after a final look. :)

**Comment by fichtner** [comment]
squashed all commits into https://github.com/opnsense/core/commit/caea4e1b10e99c5e6115c66c2690801155a8d50e

please also note https://github.com/opnsense/core/commit/9f5be0799f03af09b8f63ad3b61f2cab9ae41fae -- we don't want html in gettext strings. lots of code does it but we'll have to get rid of it eventually. :)

**Comment by fabianfrz** [comment]
I've seen this in the translation files very often so I thought it is common to do this - will do it this way in the future

can we close the related issue?

**Pull request closed without merging**